test/typescript/misc/Trans.test.tsx

```diff
@@ -126,7 +126,6 @@ describe('<Trans />', () => {
   it('should not work with object child', () => {
     assertType<React.ReactElement>(
       <Trans>
-        {/* @ts-expect-error */}
```

Comment:
No longer necessary after downgrading @types/react to v16.

Reply:
This assertion is necessary since the test checks that an error is thrown when there are object inside child computations and `allowObjectInHTMLChildren` is disabled. 

Check the line below:

```tsx
<span>This {{ var: '' }} is an error since `allowObjectInHTMLChildren` is disabled</span>
```

---

Probably that wasn't an issue with old react types. 
This is another reason to update `react` version rather than downgrading `@types/react`

Reply:
Reverted the `@types/react` downgrade since we decided to upgrade react instead.